charts/ocis/templates/_common/monitoring/servicemonitor.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.monitoring.enabled -}}
+apiVersion: monitoring.coreos.com/v1
```

Comment:
For this to work the Prometheus Operator needs to be installed on the Cluster, right? Shouldn't we mention that somewhere it the docs?